### DISCUSSION TITLE
Import queries in Annotation and Variable queries

### DISCUSSION THREAD

**Discussion by zoltanbedi** [grafana/grafana#62690] [Dashboards, Panels & Visualizations]
We have a functionality called `importFromAbstractQueries` or as it is called in the past `importQueries`. This is only called when we change the datasource in a `QueryGroup`. It would be nice if we could have this functionality in Variable queries and Annotation queries as well.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82860. Feel free to continue the discussion around this there. Thank you!
